proto/cosmos/group/v1beta1/tx.proto

```diff
@@ -279,3 +282,19 @@ message MsgExec {
 
 // MsgExecResponse is the Msg/Exec request type.
 message MsgExecResponse {}
+
+// MsgLeaveGroup is the Msg/LeaveGroup request type.
+message MsgLeaveGroup {
+  // member_address is the account address of the group member.
+  string member_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // group_id is the unique ID of the group.
+  uint64 group_id = 2;
+
+  // policy_address is the account address of group policy.
+  string policy_address = 3 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
We also need a group policy address. One group can have multiple policies.

Reply:
no we don't need this, group is the only entity managing members
if a member leaves a group, then it will be effective for all the corresponding group policies